cross_list_courses/static/cross_list_courses/js/controllers/ListController.js

```diff
@@ -60,13 +60,15 @@
                         next: '',
                         previous: '',
                     },
+                    processing: '<img src="' + window.globals.STATIC_URL + 'images/ajax-loader-large.gif" class="loading"/> Loading...',
```

Comment:
(It would avoid the extra static URL thing -- but perhaps it doesn't work well with datatables?)